api/src/schema/directory-quick-facts.graphql

```diff
@@ -927,7 +927,7 @@ extend type Stream {
       """
     )
 
-  avgBussingAttendanceThisMonth: Float!
+  avgBussingAttendanceThisMonth: Float
```

Comment:
Yes please. Because if a church does not have data for the relevant period, we should get null. A newly created church, and a church that has been on vacation for a month.

Reply:
@jaedag

Reply:
Null or 0 since we are dealing with numbers.

Reply:
> Null or 0 since we are dealing with numbers. 

Like is it null or 0?

Reply:
I feel that we should return 0 if there is no data. So that we know that null means that the query is failing clearly.

Reply:
It really helps when debugging. But you wouldn't know if you don't do such things